test/go.mod

```diff
@@ -10,7 +10,7 @@ require (
 	github.com/docker/go-connections v0.4.0
 	github.com/flatcar/ignition v0.36.2
 	github.com/go-logr/logr v1.2.3
-	github.com/onsi/ginkgo/v2 v2.5.1
+	github.com/onsi/ginkgo/v2 v2.6.0
```

Comment:
I think dependabot isn't able to update the submodules here, meaning we need to manually fix these PRs. I'll take a look and see if there's some way to fix it.